plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSharedHiveMetastoreWithViews.java

```diff
@@ -94,30 +92,30 @@ public void testReadInformationSchema()
         assertThat(query("SELECT table_schema FROM hive.information_schema.tables WHERE table_name = 'hive_table' AND table_schema='" + schema + "'"))
                 .skippingTypesCheck()
                 .containsAll("VALUES '" + schema + "'");
-        assertThat(query("SELECT table_schema FROM delta.information_schema.tables WHERE table_name = 'delta_table' AND table_schema='" + schema + "'"))
+        assertThat(query("SELECT table_schema FROM delta_lake.information_schema.tables WHERE table_name = 'delta_table' AND table_schema='" + schema + "'"))
                 .skippingTypesCheck()
                 .containsAll("VALUES '" + schema + "'");
 
         assertQuery("SELECT table_name, column_name from hive.information_schema.columns WHERE table_schema = '" + schema + "'",
                 "VALUES ('hive_table', 'a_integer')");
-        assertQuery("SELECT table_name, column_name from delta.information_schema.columns WHERE table_schema = '" + schema + "'",
+        assertQuery("SELECT table_name, column_name from delta_lake.information_schema.columns WHERE table_schema = '" + schema + "'",
                 "VALUES ('delta_table', 'a_varchar')");
     }
 
     @Test
     public void testShowTables()
     {
-        assertQuery("SHOW TABLES FROM delta." + schema, "VALUES 'hive_table', 'hive_view', 'delta_table'");
+        assertQuery("SHOW TABLES FROM delta_lake." + schema, "VALUES 'hive_table', 'hive_view', 'delta_table'");
         assertQuery("SHOW TABLES FROM hive." + schema, "VALUES 'hive_table', 'hive_view', 'delta_table'");
 
-        assertThatThrownBy(() -> query("SHOW CREATE TABLE delta." + schema + ".hive_table"))
+        assertThatThrownBy(() -> query("SHOW CREATE TABLE delta_lake." + schema + ".hive_table"))
```

Comment:
optional: Using `io.trino.plugin.deltalake.DeltaLakeQueryRunner#DELTA_CATALOG` would be also on option.

Reply:
We could even omit `delta_lake` but thought this way is a bit more explicit.